tools/walletextension/wallet_extension.go

```diff
@@ -346,3 +355,50 @@ func adjustStateRoot(rpcResp interface{}, respMap map[string]interface{}) {
 		}
 	}
 }
+
+// getStorageAtInterceptor checks method name in the request and if it is one of the methods that we use differently than an eth standard.
+// At the moment, it is only getStorageAt method. It returns response if we should use that response or nil otherwise
+func (w *WalletExtension) getStorageAtInterceptor(request *accountmanager.RPCRequest, hexUserID string) map[string]interface{} {
+	response := map[string]interface{}{}
+	// all responses must contain the request id. Both successful and unsuccessful.
+	response[common.JSONKeyRPCVersion] = jsonrpc.Version
+	response[common.JSONKeyID] = request.ID
```

Comment:
No need to initialize the response if sometimes the return is nil.

Reply:
fixed.